From f32235d81d2ab123687eded7405b0c52b49fe86b Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Sun, 9 Dec 2018 15:56:59 +0100
Subject: [PATCH 06/13] Revert "Set rlimit rtprio for cameraserver"

This reverts commit 4ae244cab4fe715fc2729e906b7dda3075fbbac3.

We need to revert this because some init/systems (Moto E5, G6) doesn't
like this instruction at all (read: this prevents boot)
---
 camera/cameraserver/cameraserver.rc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/camera/cameraserver/cameraserver.rc b/camera/cameraserver/cameraserver.rc
index a9aae0b15..fea5a1d5c 100644
--- a/camera/cameraserver/cameraserver.rc
+++ b/camera/cameraserver/cameraserver.rc
@@ -4,4 +4,3 @@ service cameraserver /system/bin/cameraserver
     group audio camera input drmrpc
     ioprio rt 4
     writepid /dev/cpuset/camera-daemon/tasks /dev/stune/top-app/tasks
-    rlimit rtprio 10 10
-- 
2.17.1

